File: Example/src/PhysCalc.java
import java.util.Scanner;

public class PhysCalc {

  Scanner s;
  double energy;
  double velocity;
  double mass;

  PhysCalc() {
    s = new Scanner(System.in);
  }

  public void getMass() {
    System.out.print("Enter the mass: ");
    mass = s.nextDouble();
    s.nextLine();
  }

  public void getVelocity() {
    System.out.print("Enter the velocity: ");
    velocity = s.nextDouble();
    s.nextLine();
  }

  public void getEnergy() {
    System.out.print("Enter the energy: ");
    energy = s.nextDouble();
    s.nextLine();
  }

  public double kinEnergy() {
    energy = 0.5 * mass * velocity * velocity;
    return energy;
  }

  public double velocity() {
    velocity = Math.sqrt((2 * energy) / mass);
    return velocity;
  }

  public double mass() {
    mass = (2 * energy) / (velocity * velocity);
    return mass;
  }

  public void calculate() {
    char choice;
    System.out.println("Enter your selection");
    System.out.println("E -- Calculate Energy");
    System.out.println("m -- Calculate Mass");
    System.out.println("v -- Calculate Velocity");
    choice = s.nextLine().charAt(0);

    switch (choice) {
      case 'e':
      case 'E':
        getMass();
        getEnergy();

        System.out.printf("The kinetic energy is: %.2f", kinEnergy());
        break;

      case 'v':
      case 'V':
        getEnergy();
        getMass();

        System.out.printf("The velocity is: %.2f", velocity());
        break;

      case 'm':
      case 'M':
        getVelocity();
        getEnergy();

        System.out.printf("The mass is: %.2f", mass());
        break;

      default:
        System.out.println("Invalid Menu option");
        break;
    }

  }

  public static void main(String[] args) {
    PhysCalc p = new PhysCalc();
    p.calculate();
  }
}

File: Example/src/PhysCalc.py
import math

class PhysCalc:

    def __init__(self):
        self.energy = 0.0
        self.velocity = 0.0
        self.mass = 0.0

    def get_mass(self):
        self.mass = float(input("Enter the mass: "))

    def get_velocity(self):
        self.velocity = float(input("Enter the velocity: "))

    def get_energy(self):
        self.energy = float(input("Enter the energy: "))

    def kin_energy(self):
        self.energy = 0.5 * self.mass * self.velocity ** 2
        return self.energy

    def calc_velocity(self):
        self.velocity = math.sqrt((2 * self.energy) / self.mass)
        return self.velocity

    def calc_mass(self):
        self.mass = (2 * self.energy) / (self.velocity ** 2)
        return self.mass

    def calculate(self):
        print("Enter your selection")
        print("E -- Calculate Energy")
        print("m -- Calculate Mass")
        print("v -- Calculate Velocity")
        choice = input().strip().lower()

        if choice == 'e':
            self.get_mass()
            self.get_velocity()
            print(f"The kinetic energy is: {self.kin_energy():.2f}")
        elif choice == 'v':
            self.get_energy()
            self.get_mass()
            print(f"The velocity is: {self.calc_velocity():.2f}")
        elif choice == 'm':
            self.get_velocity()
            self.get_energy()
            print(f"The mass is: {self.calc_mass():.2f}")
        else:
            print("Invalid Menu option")

if __name__ == "__main__":
    p = PhysCalc()
    p.calculate()
